core-bundle/contao/themes/flexible/main.css

```diff
@@ -2581,6 +2582,12 @@ fieldset.collapsed > legend {
 	background-color: var(--table-odd);
 }
 
+#picker-menu > ul {
```

Comment:
Can you please move this up to where the other `#picker-menu` definitions are (around line 2126)?

Reply:
Done